### PR TITLE
Issue 94 - Reduce margin above tokens to 40px

### PR DESCRIPTION
**Pull request by FrancescoSTL**
Reducing space between bottom of sidebar-content and top of tokens to 40px.

Issue ref: #94

### CODE REVIEW

File: data/css/all-aboard.css [hunk 1]
```diff
@@ -82,7 +86,7 @@ a:focus {
 }
 
 .sidebar-content.values footer {
-    padding-top: 40px;
+    padding-top: 0;
```
Comment (schalkneethling) [inline data/css/all-aboard.css]: As we are turning off top padding for all the values and utility sidebars, we can simplify this to one rule so, remove:
```
.sidebar-content.values footer
```
and
```
.sidebar-content.utility footer
```
and replace with just
```
.sidebar-content footer {
    padding-top: 0;
}
```

**Comment by FrancescoSTL** [comment]
Your wish is my command. r? @schalkneethling

**Comment by FrancescoSTL** [comment]
Okay now your wish is my command. r? @schalkneethling

**Comment by schalkneethling** [comment]
r+ 🎉

**Pull request merged**